test/components/previews/selection_preview.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class SelectionPreview < Lookbook::Preview
```

Comment:
**chore (blocking):** This should be a `ViewComponent::Preview` not `Lookbook::Preview`